README.md

```diff
@@ -23,6 +23,10 @@ Specialized methods manage categories and files:
    those that are no longer needed.
 - `list_files()` lists remote files, their attributes, and Unix-style path
   representations of their hierarchical position in the category tree.
+- `list_files()` List remote tax rates with their attributes.
+- `list_tax_rates()` lists remote files, their attributes, and Unix-style path
```

Comment:
I believe `list_tax_rates()` does not add a path.

Change to: 
- `list_tax_rates()` lists remote files, their attributes. ~~and Unix-style path~~

Reply:
fixed